stdlib/asm/collections/smt.masm

```diff
@@ -431,9 +507,19 @@ export.get
             hmerge assert_eqw
             # => [V, R]
         else
-            # Multiple kv-pair case
-            # TODO (fail for now)
-            push.1 assertz
+            # => [leaf_size, NV, K, R]
+
+            # We don't care about the node's hash anymore.
+            movdn.12 dropw
+            # => [K, R, leaf_size]
```

Comment:
I think we probably need the node value - otherwise we won't be able to verify that the data from the advice provider is correct (advice provider is an untrusted component and, in theory, could "lie").